ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -34,6 +39,10 @@ test.beforeAll(async () => {
   await checkBoundaryCli();
 });
 
+test.afterEach(async () => {
+  await deleteOrg(org.id);
+})
+
```

Comment:
Instead of doing this approach, what if we added a `try/finally` in a test to do the cleanup (i.e. https://github.com/microsoft/playwright/discussions/10941#discussioncomment-1813578)? That way, the cleanup is more explicit when reading a test and we remove the need of global variables. What are your thoughts on that?

Reply:
I personally like how the framework capabilities (e.g. `afterEach` method) allow to get rid of duplications in test methods and move test data creation and removal out of them. In our case, we already have a `try/finally` statement used in both tests, so reusing it for the clean up makes sense 👍